Assets/Scripts/InventoryTracker.cs

```diff
@@ -9,8 +9,10 @@ public class InventoryTracker : MonoBehaviour
     public  void Init()
     {
         _inventory.Add("Body0", 4);
-        _inventory.Add("Body1", 2);
-        _inventory.Add("Body2", 2);
+
+        _inventory.Add("Body1", 6);
+        _inventory.Add("Body2", 6);
```

Comment:
`Body1` und `Body2` sollten weniger sein als `Body0`

Reply:
Muss in #19 nochmal geändert werden.